database_management_bot.py

```diff
@@ -0,0 +1,2235 @@
+
+    if admins_collection.count_documents({"user_id": user_id}) != 0:
+        logger.info("Admin is already registered, cancelling adding process")
+
+        reply_keyboard = MAIN_REPLY_KEYBOARD
+
+        update.message.reply_text(
+            text="☑️ Ви вже пройшли авторизацію. Вас зареєстровано як адміністратора",
+            reply_markup=ReplyKeyboardMarkup(
+                reply_keyboard,
+                resize_keyboard=True,
+            ),
+        )
+        return ConversationHandler.END
+
+    contact_button = KeyboardButton(text="Надіслати контакт", request_contact=True)
+    cancel_button = KeyboardButton(text="Скасувати реєстрацію")
+    custom_keyboard = [[contact_button, cancel_button]]
+    reply_markup = ReplyKeyboardMarkup(custom_keyboard,
+                                       one_time_keyboard=True,
+                                       resize_keyboard=True,
+                                       input_field_placeholder='Реєстрація')
+
+    update.message.reply_text(
+        text='🔐 Реєстрація потрібна для забеспечення безпеки та зменшення кількості спаму'
+             '\n\n✅ *Аби зареєструватись, оберіть опцію "Надіслати контакт"*'
+             '\n\n↩️ Якщо ви не бажаєте реєструватись, оберіть опцію "Скасувати реєстрацію"'
+             '\n\n❕ Зверніть увагу \- не пройшовши авторизацію ви *не зможете* вносити зміни до бази даних',
+        parse_mode="MarkdownV2",
+        reply_markup=reply_markup,
+    )
+
+    return CONTACT
+
+
+@under_maintenance
+def get_admin_photo(update: Update, context: CallbackContext) -> int:
+    """
+    The get_admin_photo_experimental function is called when the user sends a contact.
+    It stores the phone number in context.user_data, and then asks for a selfie photo to be sent.
+
+    :param update: Update: Access the telegram api
+    :param context: CallbackContext: Send data between functions
+    :return: The next state of the concersation
+    """
+    logger.info("User send contact")
+
+    reply_keyboard = [['Скасувати реєстрацію']]
+
+    context.user_data["phone_number"] = update.message.contact.phone_number
+    logger.info("Storing admin phone number")
+
+    context.user_data["ADMIN_INFO"] = update.message.contact.to_dict()
+    logger.info("Storing admin contact info")
+
+    update.message.reply_text(
+        text='👤 *Тепер, надішліть, будь ласка, фото свого обличчя*'
+             '\n\n😃 Не лякайтесь\! Ця процедура потрібна для підвищення безпеки під час війни\.'
+             '\n\n🔒 Ми подбаємо про *конфіденційність та безпеку* ваших даних\.'
+             ' Уся інформація зберігається на *захищених серверах*\. Ми *не будемо поширювати* її\.'
+             '\n\n❕ Будь ласка, використовуйте реальну фотографію вашого обличчя\.'
+             ' Фотографуйте при хорошому освітленні та старайтесь, щоб ваше обличчя було добре видно на фото\.'
+             '\n\n❗️ Вам буде заблоковано доступ за використання несправжніх фотографій обличчя\.',
```

Comment:
It may sound better.
```suggestion
             '\n\n❗️ Для вас доступ буде заблоковано у разі використання несправжніх фотографій обличчя\.',
```

Reply:
Accepted with edits
```suggestion
             '\n\n❗️ Ваш акаунт буде заблоковано у разі використання несправжніх фотографій обличчя\.',
```